homeassistant/components/roborock/scene.py

```diff
@@ -0,0 +1,75 @@
+"""Support for Roborock scene."""
+
+from __future__ import annotations
+
+import asyncio
+from typing import Any
+
+from roborock.containers import HomeDataScene
+
+from homeassistant.components.scene import Scene as SceneEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import RoborockConfigEntry
+from .coordinator import RoborockDataUpdateCoordinator
+from .entity import RoborockEntity
+
+
+async def _setup_coordinator(
```

Comment:
get_scenes is an async function so i don't think this is correct?

Reply:
Not sure what you are referring to that is incorrect, if you mean the function name, I agree and changed it

Reply:
I was referring tot he typing -- I may have misread where the `await` was, my mistake.